app/websocket/src/main/kotlin/com/xorker/draw/websocket/message/request/dto/VoteMafiaRequest.kt

```diff
@@ -0,0 +1,5 @@
+package com.xorker.draw.websocket.message.request.dto
+
+data class VoteMafiaRequest(
+    val userId: Long,
```

Comment:
p2. 그냥 value class 사용해도 됩니다. 컴파일하면 Long 으로 바뀌어요

```suggestion
    val userId: UserId,
```